tests/test_csc.py

```diff
@@ -302,23 +303,24 @@ async def test_invalid_track_target(self) -> None:
             await self.assert_next_sample(
                 self.remote.evt_atMountState, state=AtMountState.TRACKINGDISABLED
             )
-            await self.fault_to_enabled()
+            # await self.fault_to_enabled()
 
     async def test_brake_and_drive_status_events(self) -> None:
         async with self.make_csc(initial_state=salobj.State.STANDBY):
-            # Axes start disabled.
-            for event in self.brake_events:
-                await self.assert_next_sample(event, engaged=True)
-
-            for event in self.drive_status_events:
-                await self.assert_next_sample(event, enable=False)
+            # No events get emitted since no connection has been made yet to
+            # the remote ATMCS system.
+            with pytest.raises(asyncio.TimeoutError):
+                await self.remote.evt_azimuthBrake1.next(flush=False, timeout=0.1)
```

Comment:
Please use a named constant for the timeout. This is especially important because you have a lot of these.